test/client_test.py

```diff
@@ -73,7 +73,6 @@ def test_state_event():
 
     room.name = False
     room.topic = False
-    room.aliases = False
```

Comment:
I found that when initializing aliases in `rooms.py` used empty list https://github.com/matrix-org/matrix-python-sdk/blob/00186bab2112ff6bf2a6eab20a3c465c4cf71d05/matrix_client/room.py#L43, so using `False` here doesn't look correct.